source/process/product-recipes-information-architecture.rst

```diff
@@ -0,0 +1,23 @@
+Product Recipes Information Architecture
+=========================================
+
+This document defines the product recipes information architecture. The product
+recipes might include information on:
+
+- Getting user traction for a product
+- Product installation recipes
+- Enablement of KPIs
+- Enabling user requests
+
+Organizing Product Recipes
+--------------------------
+
+Below are the things you should follow for organizing product recipes.
+
+- Write as per the document present in `administration/product-recipes-template.rst`.
+- You should write all the product recipes in the `product-recipes` directory.
+- If you create any new file you should add the file in the table of contents
+  document.
```

Comment:
@tapaswenipathak Can we adjust the language to be more active?  Changing language from "you should..." to "write all products recipes in the 'product-recipes' directory." (one example)